flake8_quotes/__init__.py

```diff
@@ -68,16 +94,22 @@ def get_quotes_errors(self, file_contents):
                 # ignore non strings
                 continue
 
-            if not token.string.startswith(self.quotes['bad_single']):
-                # ignore strings that do not start with our quotes
+            if token.string[0:3] in self.MULTILINE_QUOTES_STRINGS:
+                if self.multiline_quotes is None or token.string.startswith(self.multiline_quotes['bad']):
+                    continue
+
+            if not token.string.startswith(self.inline_quotes['bad']):
+                # ignore strings that do not start with our quote
                 continue
 
-            if token.string.startswith(self.quotes['bad_multiline']):
-                # ignore multiline strings
+
+
```

Comment:
The linter will prob complain about this but there's too much whitespace here =/